inc/update.class.php

```diff
@@ -434,11 +434,17 @@ public function doUpdates($current_version = null) {
 
          case "9.3.2":
          case "9.3.3":
-         case GLPI_PREVER:
             include_once("{$updir}update_93_94.php");
             update93to94();
             break;
 
+         case "9.4":
```

Comment:
There is no "9.4" release. I've rebased and did the change; but I'm not able to push on your branch...

Reply:
see https://github.com/glpi-project/glpi/compare/9.4/bugfixes...trasher:fix_profile_search_option